.gitignore

```diff
@@ -5,6 +5,7 @@ package-lock.json
 yarn.lock
 /examples/**/.svelte
 /examples/**/build
+/sandbox
```

Comment:
does this make sense to add?

Reply:
i dunno. it's been really helpful to have a sandbox app that's linked to the workspace, but as we saw you can't avoid that stuff showing up in the lockfile. maybe i should just do that stuff in a completely separate repo that links to the packages in here, i guess i'll delete this